### PR TITLE
Swap out memcached backend

### PR DESCRIPTION
**Pull request by bderusha**
Add static target & update docker-compose and prod settings

@clintonb please review

FYI @waheedahmed @awais786

### CODE REVIEW

File: requirements/local.txt [hunk 1]
```diff
@@ -9,4 +9,4 @@ transifex-client==0.11
 git+https://github.com/edx/i18n-tools.git@v0.1.4#egg=i18n_tools==0.1.4
 
 # docker devstack
-docker-compose >=1.5.1,<1.6.0
+docker-compose==1.7.1
```
Comment (clintonb) [inline requirements/local.txt]: `>=1.7.1,<2.0.0`

File: course_discovery/settings/base.py [hunk 2]
```diff
@@ -354,9 +349,4 @@
 HAYSTACK_SIGNAL_PROCESSOR = 'haystack.signals.RealtimeSignalProcessor'
 
 
-COMPRESS_PRECOMPILERS = (
```
Comment (clintonb) [inline course_discovery/settings/base.py:357]: Isn't this needed for development?

**Comment by clintonb** [comment]
👍

File: course_discovery/settings/base.py (hunk 2 above)
Comment (bderusha) [inline course_discovery/settings/base.py:357]: This is a dupe https://github.com/edx/course-discovery/pull/209/files#diff-df3521e313a8eca75a92577b339f5104R142

**Pull request merged**